libs/pavexc/src/compiler/codegen.rs

```diff
@@ -478,6 +478,7 @@ fn get_request_dispatcher(
     syn::parse2(quote! {
         async fn route_request(
             request: #http::Request<#hyper::body::Incoming>,
+            connection_info: Option<::pavex::connection::ConnectionInfo>,
```

Comment:
```suggestion
            connection_info: Option<#pavex::connection::ConnectionInfo>,
```

Pavex might be renamed in the manifest for the generated crate (e.g. there might be multiple versions around). The value in the `pavex` variable is guaranteed to be the right crate name for this purpose.

Reply:
https://github.com/LukeMathWalker/pavex/pull/169/commits/2e5fbba06c46bc1658022d7b05f16ea9103b4cc6